config.py

```diff
@@ -288,8 +290,6 @@ def cdns_loaded_from_database(cls):
 
     @classmethod
     def get(cls, key, default=None):
-        if cls.instance is None:
-            raise ValueError("No configuration object loaded!")
```

Comment:
This error seems like it could still be useful, in case we break the code that loads the configuration.